### ISSUE TITLE
Add direct memory address modification instead of Game Genie codes

### ISSUE THREAD

**Issue by calvincorrieri** [rileytestut/Delta#416]
Game Genie codes are unable to modify memory addresses below 0x8000. Some games store values like number of lives below 0x8000, making it completely impossible to alter the values using Game Genie codes. A lot of emulators however have cheat menus where you can directly alter memory values. You input the memory address, the desired value, and a compare value if you want. This is infinitely better than the cryptic and handicapped Game Genie system.

TL;DR: Please add direct memory address modification cheats because Game Genie is awful.
